polylogue/cli/render.py

```
@@ -73,11 +76,21 @@ def _truthy(val: str) -> bool:
         attachment_ocr=getattr(args, "attachment_ocr", False) or _truthy(render_prefs.get("--attachment-ocr", "false")) if render_prefs else getattr(args, "attachment_ocr", False),
         sanitize_html=getattr(args, "sanitize_html", False),
+        meta=meta,
     )
     if getattr(args, "max_disk", None):
         projected = len(inputs) * 5 * 1024 * 1024  # rough 5MiB per file heuristic
         from ..util import preflight_disk_requirement
 
+        if not json_output:
+            try:
+                free_bytes = shutil.disk_usage(Path.cwd()).free
+            except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)